### PR TITLE
Ci maintenance

### PR DESCRIPTION
**Pull request by safl**
The Windows build started failing due to external changes beyond our control. While the build on next passed, it has consistently failed on PRs since. This PR includes CI maintenance and a work-in-progress fix for the Windows build regression.

### CODE REVIEW

File: docs/toolchain/freebsd/index.rst [hunk 1]
```diff
@@ -34,3 +34,34 @@ Then go ahead and configure, build and install using ``meson``:
...
 
 
+FreeBSD (14)
+~~~~~~~~~~~~
+
+Install the required toolchain and libraries, with sufficient system privileges
+(e.g. as ``root`` or with ``sudo``), by executing the commands below. You can
+run this from the root of the **xNVMe** by invoking::
+
+  sudo ./xnvme/toolbox/pkgs/freebsd-14.sh
```
Comment (NaddiNadja) [inline docs/toolchain/freebsd/index.rst]: If it's from "the root of xNVMe", there shouldn't be a `./xnvme` in path, right?

**Review by NaddiNadja — COMMENTED** [review]
Just one question, otherwise it looks good to me now that the tests are green :)

File: docs/toolchain/freebsd/index.rst (hunk 1 above)
Comment (safl) [reply docs/toolchain/freebsd/index.rst]: Nice catch!

Comment (NaddiNadja) [reply docs/toolchain/freebsd/index.rst]: You still have 
```
You can run this from the **xNVMe** repository by invoking::

  sudo ./xnvme/toolbox/pkgs/freebsd-13.sh
```

If it's "from" the xNVMe repo, shouldn't the command just be `sudo ./toolbox (...)` or is this some kind of convention I don't know of? :)

Comment (safl) [reply docs/toolchain/freebsd/index.rst]: yeah. Do you have suggestion on how to fix the phrasing without changing the command?

Comment (safl) [reply docs/toolchain/freebsd/index.rst]: How about this...

**Review by NaddiNadja — DISMISSED** [review]
lgtm :)

File: docs/toolchain/freebsd/index.rst (hunk 1 above)
Comment (NaddiNadja) [reply docs/toolchain/freebsd/index.rst]: i like that one! :)

**Comment by safl** [comment]
Gave it another rewording :)

**Comment by safl** [comment]
Forcing this in, to unblock the pipeline.

**Pull request merged**